cypress/integration/assurance_visits.spec.js

```diff
@@ -0,0 +1,31 @@
+describe("Manage Assurance Visits", () => {
+    beforeEach(() => {
+        cy.setCookie("Other", "other");
+        cy.setCookie("XSRF-TOKEN", "abcde");
+        cy.visit("/supervision/deputies/3/assurance-visits");
+    });
+
+    describe("Header", () => {
+        it("shows title and button", () => {
+            cy.get(".govuk-main-wrapper > header").contains("Assurance visits");
+            cy.get(".govuk-button").contains("Add a visit");
+        });
+    });
+
+    describe("Main content", () => {
+        it("should display assurance visit main content", () => {
+            cy.get(":nth-child(5) > .govuk-grid-column-one-half > #assurance-visit-details > :nth-child(1) > .govuk-summary-list__key").contains("Requested date");
```

Comment:
Given you have `#assurance-visit-details` everything before that is irrelevant to the CSS selector as the id should be unique on the page.

Reply:
done